src/main/java/spireMapOverhaul/zones/thieveshideout/monsters/BanditLieutenant.java

```diff
@@ -0,0 +1,141 @@
+package spireMapOverhaul.zones.thieveshideout.monsters;
+
+import basemod.abstracts.CustomMonster;
+import com.badlogic.gdx.math.MathUtils;
+import com.esotericsoftware.spine.AnimationState;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateShakeAction;
+import com.megacrit.cardcrawl.actions.animations.AnimateSlowAttackAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.*;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.status.Dazed;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import com.megacrit.cardcrawl.powers.DrawReductionPower;
+import com.megacrit.cardcrawl.powers.VulnerablePower;
+import com.megacrit.cardcrawl.vfx.combat.SmokeBombEffect;
+import spireMapOverhaul.SpireAnniversary6Mod;
+
+public class BanditLieutenant extends CustomMonster {
```

Comment:
I didn't use `AbstractSMOMonster` for these monsters (or the `PhasedEvent` stuff for the events) because I already have good setups for the older style of making monsters/events and don't find the new style any faster.